releasenotes/notes/fix-bip-mapper-requirements-930c7dcd5a3b82ba.yaml

```
@@ -0,0 +1,10 @@
+---
+fixes:
+  - |
+    Fix ``pip install qiskit-terra[all]`` on Python 3.6 and 3.9.  On these
+    versions, the ``bip-mapper`` target constraints cannot be satisfied due to
+    the ``cplex`` package only supporting Python 3.7 and 3.8.  Requesting the
+    ``all`` target on these versions will now not attempt to install the
+    unsolvable dependency, though you can still attempt to install the
+    ``bip-mapper`` extra directly if you want to force pip to try and resolve
+    the package.
```

Just some wording tweaks to try and make it a bit clearer:

```suggestion
    Fixed ``pip install qiskit-terra[all]`` so that it will successfully install when
    running on Python 3.6 and 3.9.  Previously, these Python versions would
    fail to install the ``all`` target because of the the ``bip-mapper`` extra
    requirements could not be satisfied due to the ``cplex`` package only
    supporting Python 3.7 and 3.8.  Requesting the ``all`` target on these
    Python versions will now no longer attempt to install the unsolvable
    dependency (ie ``cplex`` and ``docplex`` will not longer be attempted to
    installed). The ``bip-mapper`` extra will still attempt to install the ``cplex``
    and ``docplex`` packages on Python 3.6 and 3.9 (and likely fail as
    ``cplex isn't available).
```

---

Done manually to fix a typo and an rst bug.